en/docs/learn/consume-api/discover-apis/search.md

```diff
@@ -74,6 +74,11 @@ You can search for APIs in the API Publisher or Developer Portal in the followin
 <p>A property can be defined to an API at the time it is created or later. There can be APIs without properties as this parameter is optional. <strong><br />
 </strong></p></td>
 </tr>
+<tr class="even">
+<td>By API category</td>
+<td><p><strong>api-category:xxxx</strong> . For example, api-category:test</p>
+<p>An API category can be used to group APIs based on any property. API categories can be defined from the Admin Portal and can be assigned to an API from the Design Configurations section of the Publisher Portal.</p></td>
```

Comment:
"Any property" implies that API properties are bound with API categories. Shall we reword this? "An API category can be used to group APIs"

Reply:
According to our offline conversation rephrased property to criteria https://github.com/wso2/docs-apim/pull/2615/commits/fe3422d92d36fc2dc410304041a563ff57dcfd06